operators/src/processing/raster_vector_join/points.rs

```diff
@@ -129,17 +138,23 @@ impl RasterPointJoinProcessor {
     }
 }
 
-struct JoinerState {
-    points: MultiPointCollection,
+struct JoinerState<G, C> {
+    covered_pixels: C,
     aggregator: TypedAggregator,
+    g: PhantomData<G>,
 }
 
-struct PointRasterJoiner {
-    state: Option<JoinerState>,
+struct PointRasterJoiner<G, C> {
```

Comment:
rename, and rename files: non_aggregated, aggregated

Reply:
62098aac28fc3cbeb8085333658938d7422428a5
3d11868877a9d31d159974c595abace45a68e160